nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -42,6 +42,29 @@
   </itemizedlist>
  </section>
 
+ <section xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="sec-release-19.09-incompatibilities">
+  <title>Backward Incompatibilities</title>
+
+  <para>
+   When upgrading from a previous release, please be aware of the following
+   incompatible changes:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     Buildbot master no longer supports Python 2, as support was dropped
```

Comment:
```suggestion
     Buildbot no longer supports Python 2, as support was dropped
```

Reply:
Buildbot worker still supports Python 2.

Reply:
Well, even though that is technically true, there is really no reason to use it with Python 2, so I might as well remove that to make it less confusing.

Reply:
> so I might as well remove that to make it less confusing

Either that or use "build master" as it's called on http://buildbot.net/.